tests/by-util/test_hashsum.rs

```diff
@@ -180,6 +180,21 @@ fn test_check_b2sum_length_option_0() {
         .stdout_only("testf: OK\n");
 }
 
+#[test]
+fn test_check_b2sum_length_duplicate() {
+    let scene = TestScenario::new(util_name!());
+    let at = &scene.fixtures;
+
+    at.write("testf", "foobar\n");
+
+    scene
+        .ccmd("b2sum")
+        .arg("--length=123")
+        .arg("--length=128")
+        .succeeds()
+        .stdout_contains("cae66941d9efbd404e4d88758ea67670");
+}
```

Comment:
This test is misleading: it implies the expected checksum is somehow related to the `testf` file. But that's not the case. The `at` variable could be removed and the test would still pass ;-)

Instead the test implicitly tests the following: `printf "" | b2sum --length=123 --length=128`. So I would either pass the filename with `arg` to `b2sum` and adapt the checksum, or pipe in `""` and remove the `at` variable.

Reply:
well spotted